proto/kv/v1/kv.proto

```diff
@@ -15,17 +15,33 @@ option csharp_namespace = "Nitric.Proto.KeyValue.v1";
 service KeyValue {
```

Comment:
Let's cherrypick out the kv.proto changes, either on a new branch or on this branch

Reply:
I was thinking about rollback those changes to KV, leave it as is. Removing the kv is also an option to discuss at this early stage.